src/main/java/au/edu/wehi/idsv/VariantCaller.java

```diff
@@ -32,7 +32,7 @@ public VariantCaller(ProcessingContext context, List<SAMEvidenceSource> samEvide
 		this.samEvidence = samEvidence;
 		this.assemblyEvidence = assemblyEvidence;
 	}
-	public void callBreakends(File vcf, ExecutorService threadpool) throws IOException {
+	public void callBreakends(File vcf, ExecutorService threadpool, int job_index, int job_nodes) throws IOException {
```

Comment:
@mayalevy - is this your change? Or it is coming from my code? Asking because in my code it was sort of for debugging and it requires you to fix tests. Maybe you want to drop it? Or at least make it so the test do not break?

Reply:
@ilyasoifer Not mine at all. I'll drop that